source/extensions/access_loggers/http_grpc/grpc_access_log_impl.cc

```diff
@@ -43,13 +43,32 @@ void GrpcAccessLoggerImpl::LocalStream::onRemoteClose(Grpc::Status::GrpcStatus,
 }
 
 GrpcAccessLoggerImpl::GrpcAccessLoggerImpl(Grpc::RawAsyncClientPtr&& client, std::string log_name,
+                                           std::chrono::milliseconds buffer_flush_interval_msec,
+                                           size_t buffer_size_bytes, Event::Dispatcher& dispatcher,
```

Comment:
nit: s/size_t/uint64_t for buffer size (prefer explicit integer sizes in Envoy). Same elsewhere.